### ISSUE TITLE
CocoaPod?

### ISSUE THREAD

**Issue by kav** [MailCore/MailCore#22]
Any thoughts on enabling installation via CocoaPods. Happy to help but want to make sure you're interested first.

**Comment by jwilling** [comment]
Yes, I've already discussed this with @mronge, and he's in favor of getting CocoaPods support in here. The dependencies are nasty to deal with, though.

**Comment by mronge** [comment]
I would LOVE to have CocoaPod support. I don't have time to do it myself right now but patches appreciated.

Also, the latest version of LibEtPan (https://github.com/dinhviethoa/libetpan/) is slightly newer than the one MailCore is using. If you update MailCore to the latest version (no API changes required, just messing with project files) OpenSSL is no longer a dependency. Maybe that makes it easier?

**Comment by DougPuchalski** [comment]
+1

**Comment by bparousis** [comment]
+1

**Comment by rwyland** [comment]
Any updates on CocoaPods?  Also, I noticed that the dependencies for MailCore were OpenSSL, CyrusSASL, and LibetPan; has any of this changed that would make enabling CocoaPods easier?

Thanks

**Comment by joejcon1** [comment]
+1

**Comment by fabiopelosin** [comment]
Hi, the dependencies are not an issue anymore and MailCore appears to work with the following podspec:

```
Pod::Spec.new do |s|
  s.name         = "MailCore"
  s.version      = "0.0.1"
  s.summary      = "A Mac/iOS framework for working with the e-mail protocols IMAP and SMTP."
  s.homepage     = "https://github.com/mronge/MailCore"
  s.license      = "BSD"
  s.author       = { "Matt Ronge" => "mronge@mronge.com" }
  s.source       = { :git => "https://github.com/mronge/MailCore.git" }
  s.platform     = :osx
  s.source_files = "Source"
  s.dependency 'libetpan'
end
```

As there is already a tag, I need a new one to push this to the CocoaPods specs repo.

**Comment by iljaiwas** [comment]
This MailCore podspec seems to be not working at all, mainly because the 'libetpan' podspec seems to be a complete mess. I'm trying come up with a working MailCore podspec, but it's indeed difficult. I could use some help.

As a first step, here is a revised 'libetpan' pod spec which seems to include all the necessary source code in the Xcode project created by CocoaPods:
https://github.com/iljaiwas/Specs/blob/master/libetpan/0.0.1/libetpan.podspec

**Comment by paulyoung** [comment]
+1
